bin/cylc-insert

```diff
@@ -43,39 +43,47 @@ if '--use-ssh' in sys.argv[1:]:
 import cylc.flags
 from cylc.prompt import prompt
 from cylc.network.suite_command import SuiteCommandClient
-from cylc.CylcOptionParsers import cop, multitask_usage
+from cylc.CylcOptionParsers import cop
 
 
 def main():
     parser = cop(
-        __doc__ + multitask_usage,
-        pyro=True, multitask=True,
+        __doc__, pyro=True, multitask=True,
         argdoc=[
             ("REG", "Suite name"),
-            ('MATCH', 'Task or family name matching regular expression'),
-            ('CYCLE_POINT', 'Cycle point (e.g. date-time or integer)'),
-            ('[STOP_POINT]', 'Optional stop cycle point for inserted task.')])
```

Comment:
Actually an inserted task proxy with a "stop point" gets removed from the pool after the point, not held as if it passed the suite final cycle point.  So it seems to me `--stop-point` or perhaps `--remove-point` would be appropriate, but not `--hold-point`.